src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,71 @@
+import * as SecureStore from 'expo-secure-store';
+
+import { device, secrets } from '../config';
+import * as appJson from '../../app.json';
+
+const namespace = appJson.expo.slug as keyof typeof secrets;
+
+export enum PushNotificationStorageKeys {
+  PUSH_TOKEN = 'PUSH_TOKEN',
+  IN_APP_PERMISSION = 'IN_APP_PERMISSION',
+}
+
+export const handleIncomingToken = async (token?: string) => {
```

Comment:
can you please add a comment for describing the behavior here for our future selfs.

Reply:
added comment in 079ae03f862b1547eecbaa6eb998264f235dac55